extensions/blocks/membership-button/membership-button.php

```diff
@@ -7,7 +7,11 @@
  * @package Jetpack
  */
 
-require_once JETPACK__PLUGIN_DIR . '/modules/memberships/class-jetpack-memberships.php';
+if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+	require_once WP_CONTENT_DIR . '/mu-plugins/memberships/class-jetpack-memberships.php';
+} elseif ( Jetpack::is_active() ) {
+	require_once JETPACK__PLUGIN_DIR . '/modules/memberships/class-jetpack-memberships.php';
```

Comment:
Would it be an option to place the file there on wpcom as well?

Reply:
I did not realize we do that, I assumed all JP modules have to be in the mu-plugins.
I'll happily move it!

Reply:
Moved! 🚀